src/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+
+### Prerequisites
+
+* **Git:** You should have Git installed on your system. If not, download and install it from the [official Git website](https://git-scm.com/).
+* **GitHub Account:** You'll need a GitHub account to fork the repository and submit pull requests. 
+* **Python:** You should have a good understanding of Python programming. If you're new to Python, there are many resources available online to get you started. You can find the official documentation on the [Python website](https://www.python.org/doc/).
+* **gRPC Concepts:** Familiarize yourself with the basics of gRPC, including concepts like protocol buffers, services, clients, and servers. Refer to the [gRPC documentation](https://grpc.io/docs/) for an overview.
+* **Bazel:** Bazel is the build system used for gRPC. To install it, follow the instructions for your operating system on the [Bazel website](https://bazel.build/install).
+
+### Steps to Contributing to gRPC Python
+
+1. **Find an Issue:**
+   * **Browse Open Issues:** Look for issues labeled "[help wanted](https://github.com/grpc/grpc/issues?q=is%3Aopen+label%3A%22disposition%2Fhelp+wanted%22)" on the [gRPC Python issue tracker](https://github.com/grpc/grpc/issues?q=is%3Aissue+is%3Aopen+label%3Alang%2Fpython+sort%3Aupdated-desc).
+   * **Ask for Help:** If you're unsure where to start or need clarification on an issue, feel free to ask questions on the [gRPC forum](https://groups.google.com/g/grpc-io).
+2. **Get a Copy of the gRPC Python Development Version:**
+   * **Fork:**  Click the "Fork" button on the top right of the [gRPC repository page](https://github.com/grpc/grpc) to create a copy of the repository under your GitHub account.
+   * **Clone your fork:**
+   ```git clone https://github.com/<your-username>/grpc.git```
+   * **Create a Branch:** Make your changes on a new branch: 
+   ```git checkout -b my-feature-branch```
+3. **Setting up Your Local System for Development and Testing:**
+   * Create a new virtual environment by running:```python -m venv ~/.virtualenvs/grpc-python```
+   * Activate the environment: ```source ~/.virtualenvs/grpc-python/bin/activate```
+   * Install Dependencies: ```pip install -r requirements.txt```
+4. **Run Tests:**
+   * Before making any changes, run the existing test suite to ensure your environment is set up correctly:
+   * **Using Bazel (Recommended):**
+     * To run a single unit test:
+       ```bash
+       bazel test --cache_test_results=no --test_output=streamed --runs_per_test=1 --test_timeout=50 "//src/python/grpcio_tests/tests/unit:_abort_test" 
+       ```
+     * To execute all unit tests for Python:
+       ```bash
+       bazel test --cache_test_results=no --test_output=streamed --runs_per_test=1 --test_timeout=50 "//src/python/..." 
+       ```
+   * **Using Provided Scripts (Alternative):**
+     * **Install Python Modules:**
```

Comment:
We should keep format the same, either use **Bold** case above or remove bold case here.

Reply:
About this, I mean remove `**` used in `Install Python Modules:` and `Run Tooling Tests:`

Reply:
removed ** used in Install Python Modules: and Run Tooling Tests: